server/views/pages/activities/manage-allocations/choose-end-date-option.njk

```diff
@@ -0,0 +1,53 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set pageTitle = "Allocation choose end date option - Activities - DPS" %}
+{% set pageId = 'allocation-choose-end-date-option-page' %}
+{% set jsBackLink = true %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <span class="govuk-caption-xl">{{ allocateJourney.activity.name }}</span>
```

Comment:
This looks a bit big on the page, maybe try govuk-caption-l?
i think UCD are wanting most titles to be L instead of XL now